windows-build/Dockerfile

```diff
@@ -3,13 +3,15 @@ FROM archlinux/archlinux:base-devel
 ENV MAKEFLAGS -j8
 ENV CGO_ENABLED 1
 
-RUN pacman -Syu --noconfirm &&\
-	pacman -S --noconfirm git go multilib-devel &&\
-	useradd -m builder &&\
-	git clone --depth 1 -b go-cgo-gcc-parallelism-1.20.3 https://github.com/Evengard/go &&\
-	cd go/src && ./make.bash && pacman -R --noconfirm go &&\
-	echo "builder ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/builder &&\
-	su builder -c "cd &&\
+RUN echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist" >> /etc/pacman.conf
```

Comment:
Why is this file changing? @Evengard is this something standard on Arch?

Reply:
When i tried to build this Docker as is, it was failing to install multilib-devel package. I looked up the problem on the official Arch Linux documentation site and read that adding this repository will give the access.
I did just that and now package is installing properly and Windows build is possible.